lms/static/sass/_developer.scss

```diff
@@ -79,4 +79,140 @@
   .global-new, #global-navigation {
     display: none;
   }
+
+  // Copied from _pagination.scss in cms
+  .pagination {
```

Comment:
Ah the LMS styles rear their ugly heads! Can you let me know when and where to take a look (this PR, a followup of yours, or a separate PR of my own)?